react/src/components/Widget/WidgetContainer.tsx

```diff
@@ -144,10 +144,14 @@ export const WidgetContainer: React.FunctionComponent<WidgetContainerProps> =
       (transaction: Transaction) => {
         if (sound && !hideToasts) txSound.play().catch(() => {});
 
-        const receivedAmount = new BigNumber(transaction.amount);
+        const { 
+          amount: transactionAmount, 
+          opReturn, 
+          hash, 
+          paymentId: transactionPaymentId } = transaction;
+        const receivedAmount = new BigNumber(transactionAmount);
 
         const currencyTicker = getCurrencyTypeFromAddress(to);
-
         if (!hideToasts)
           // TODO: This assumes only bch
```

Comment:
Is this an old comment or does it still have relevance?

Reply:
I believe this was added just for testing

Reply:
Let's remove it while we're in here.